blog/2020-07-20.adoc

```diff
@@ -0,0 +1,186 @@
+---
+layout: blog
+title: Anchored keys - scaling up a cluster without transferring values
+date: '2020-07-20T12:00:00.000-00:00'
+author: Dan Berindei
+tags: [ "anchored keys", "state transfer" ]
+---
+
+= Introducing the Anchored Keys module
+
+{brandname} version 11 introduces an experimental module that allows scaling up a cluster
```

Comment:
Don't use {brandname} for blog posts

Reply:
Sorry about that, and thanks for making the change yourself!